### ISSUE TITLE
Search: Improvements and changes to ResultList for stories, exhibitions and events

### ISSUE THREAD

**Issue by melanierogan** [wellcomecollection/wellcomecollection.org#8825]
- Summary - to caption
- Stories to include everything on /stories (also forms part of 'Overview' query ticket) -> bring in labels
- Contributors/roles (as per chat https://wellcome.slack.com/archives/C02ANCYL90E/p1667401871331959)
- Part of? Investigate

**Comment by melanierogan** [comment]
Contributors:
Jonathan Tweed
  [6 days ago](https://wellcome.slack.com/archives/C02ANCYL90E/p1667403493626869?thread_ts=1667401871.331959&cid=C02ANCYL90E)

for contributors, there are two options - either the first only or or all, separated with commas i think
no roles necessary
if we feel the need to credit an artist, we should probably include here
so that means no roles, but names in the same order as on the story itself

**Comment by melanierogan** [comment]
Circling back to this so we can find the info when we go searching for it, contributors as returned from Prismic as a graphQL query. Using [this article](https://wellcomecollection.org/articles/Y1FrCBEAAA1CDlme) as an example:

This shows you the relevant part of the query for contributors:
        ````contributors {
          contributor {
            ...on People {
              name
              
            }
            ...on EditorialContributorRoles {
              title
            }
          }
        }
this shows you the response
"contributors": [
    {
     "contributor": {
      "name": "Riaz Phillips"
     }
    },
    {
     "contributor": {
      "name": "Anna Keville Joyce"
     }
    }
   ]
```

If you don’t want to click through to the story to find out how that looks:
```Words by Riaz Phillips | artwork by Anna Keville Joyce```

TL;DR - the way we query Prismic (for search with graphql) gets us all contributors including artists, and the way i’m doing it right now does it without returning their defined roles. This seems to fit the bill for what we want right now (edited) 

:bulb: but future Mel, we might want roles!
:tada:  no problem, you can add the below to the graphql query
          ```role {
            ...on EditorialContributorRoles {
              title
            }
          }
```

adding that to the query gives you this
```"contributors": [
              {
                "contributor": {
                  "name": "Riaz Phillips"
                },
                "role": {
                  "title": [
                    {
                      "type": "heading1",
                      "text": "Author",
                      "spans": []
                    }
                  ]
                }
              },
              {
                "contributor": {
                  "name": "Anna Keville Joyce"
                },
                "role": {
                  "title": [
                    {
                      "type": "heading1",
                      "text": "Artist",
                      "spans": []
                    }
                  ]
                }
              }
            ]```

**Comment by rcantin-w** [comment]
Very entertaining read 😄 

I'm guessing what you've shown is the response we get from Prismic but we'll transform it to something else. If I understand correctly from Jonathan's comments, we still just want it to display as

`Riaz Phillips, John Smith, Anna Keville Joyce`

so for now, just a list of strings for the above would do, the order being determined in the transformer?

**Comment by melanierogan** [comment]
>so for now, just a list of strings for the above would do, the order being determined in the transformer?
ok, i'll make sure these are strings and let the transformer determine the order
